test/models/book_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class BookTest < ActiveSupport::TestCase
+  # test "the truth" do
```

Comment:
Think it's worth testing the models too, this looks useful? https://semaphoreci.com/community/tutorials/how-to-test-rails-models-with-rspec